tajo-core-tests/src/test/java/org/apache/tajo/engine/query/TestTablePartitions.java

```diff
@@ -1340,411 +1327,12 @@ public final void testDuplicatedPartitions() throws Exception {
-    res.close();
-
-    // LessThan and GreaterThan
-    res = executeString("SELECT * FROM " + tableName
-      + " WHERE key > cast('00:00:00' as time) and " +
-      "key < cast('12:10:00' as time) order by col1, col2, key desc");
-
-    expectedResult = "col1,col2,key\n" +
-      "-------------------------------\n" +
-      "1,1,11:20:40\n";
-
-    assertEquals(expectedResult, resultSetToString(res));
-    res.close();
-
-    // Between
-    res = executeString("SELECT * FROM " + tableName
-      + " WHERE key between cast('11:00:00' as time) " +
-      "and cast('13:00:00' as time) order by col1, col2, key desc");
-
-    expectedResult = "col1,col2,key\n" +
-      "-------------------------------\n" +
-      "1,1,11:20:40\n" +
-      "2,2,12:10:20\n" +
-      "3,2,12:10:30\n";
-
-    assertEquals(expectedResult, resultSetToString(res));
-    res.close();
-
-    executeString("DROP TABLE " + tableName + " PURGE").close();
-    res.close();
-  }
-
-  @Test
-  public final void testDatabaseNameIncludeTableName() throws Exception {
-    executeString("create database test_partition").close();
-
-    String databaseName = "test_partition";
-    String tableName = CatalogUtil.normalizeIdentifier("part");
-
-    if (nodeType == NodeType.INSERT) {
-      executeString(
-        "create table " + databaseName + "." + tableName + " (col1 int4, col2 int4) partition by column(key float8) ");
-
-      assertTrue(catalog.existsTable(databaseName, tableName));
-      assertEquals(2, catalog.getTableDesc(databaseName, tableName).getSchema().size());
-      assertEquals(3, catalog.getTableDesc(databaseName, tableName).getLogicalSchema().size());
-
-      executeString(
-        "insert overwrite into " + databaseName + "." + tableName + " select l_orderkey, l_partkey, " +
-          "l_quantity from lineitem");
-    } else {
-      executeString(
-        "create table "+ databaseName + "." + tableName + "(col1 int4, col2 int4) partition by column(key float8) "
-          + " as select l_orderkey, l_partkey, l_quantity from lineitem");
-    }
-
-    TableDesc tableDesc = catalog.getTableDesc(databaseName, tableName);
-    verifyPartitionDirectoryFromCatalog(databaseName, tableName, new String[]{"key"},
-      tableDesc.getStats().getNumRows());
-
-    ResultSet res = executeString("select * from " + databaseName + "." + tableName + " ORDER BY key");
-
-    String result = resultSetToString(res);
-    String expectedResult = "col1,col2,key\n" +
-      "-------------------------------\n" +
-      "1,1,17.0\n" +
-      "1,1,36.0\n" +
-      "2,2,38.0\n" +
-      "3,2,45.0\n" +
-      "3,3,49.0\n";
-    res.close();
-    assertEquals(expectedResult, result);
-
-    executeString("DROP TABLE " + databaseName + "." + tableName + " PURGE").close();
-    executeString("DROP database " + databaseName).close();
-  }
-
-  @Test
-  public void testAbnormalDirectories()  throws Exception {
+  public void testExternalTable()  throws Exception {
```

Comment:
This test includes testing on managed tables as well as external tables. Please change the name.